src/index.js

```diff
@@ -2,21 +2,22 @@ import prettify from './helpers';
 import { log } from './debug';
 
 export default (options = { schema, mode, indent }) => {
+  const customErrorToText = (error, data) => error.print(schema, data).join('\n');
+  const customErrorToStructure = (error, data) => error.getError(schema, data);
+
   const {
     schema,
     mode = 'print',
     indent = 2,
   } = options;
 
   return (data, errors) => {
-    const customErrorToText = (error) => error.print(schema, data).join('\n');
-    const customErrorToStructure = (error) => error.getError(schema, data);
     const customErrors = prettify(errors, indent);
 
     if (mode === 'print') {
-      log(customErrors.map(customErrorToText).join());
+      log(customErrors.map((error) => customErrorToText(error, data)).join());
     } else {
-      return customErrors.map(customErrorToStructure);
+      return customErrors.map((error) => customErrorToStructure(error, data));
```

Comment:
My bad 🤦‍♂️. I missed `data`. You can revert this commit.

Reply:
6f676ef